src/crypto/OlmDevice.js

```diff
@@ -219,38 +250,61 @@ OlmDevice.prototype._storeAccount = function(txn, account) {
 
 /**
  * extract an OlmSession from the session store and call the given function
+ * The session is useable only within the callback passed to this
```

Comment:
usable

Reply:
or not. my bad.

Reply:
i discovered yesterday that i've been saying "unless if" my whole life rather than just "unless". very sad.